### ISSUE TITLE
Naming convention for backtesting methods

### ISSUE THREAD

**Issue by KishManani** [JoaquinAmatRodrigo/skforecast#613]
Hello! Loving the package so far. I was reading the backtesting documentation [here](https://skforecast.org/0.11.0/user_guides/backtesting). I think some of the naming conventions contradict what I've found in a [review paper](https://link.springer.com/article/10.1007/s10618-022-00894-5) on forecasting evaluation. I was wondering whether there is wider disagreement on terminology in the field or whether this is potentially a typo in the documentation? Perhaps there is ambiguity in which origin we're referring to.

Here is an example. In the skforecast documentation the following is referred to as being fixed origin:

<img width="716" alt="image" src="https://github.com/JoaquinAmatRodrigo/skforecast/assets/30973056/bd2a6314-6d26-46be-84e0-25254afda89e">

Whereas in the paper this setup is referred to as a rolling origin:
<img width="705" alt="image" src="https://github.com/JoaquinAmatRodrigo/skforecast/assets/30973056/590c0a3f-6dc5-425e-98d7-e012a8a405b8">

Thanks again!
Kishan

**Comment by JavierEscobarOrtiz** [comment]
Hello @KishManani 

Thanks for opening the issue. If I understand correctly, it seems like the image on the right you provided with "rolling origin" is talking about the end of the training set or the origin of the validation set, **_"the forecast origin rolls forward and the forecast horizon is constant"_**.

In the user guide, "fixed origin" and "rolling origin" refer to the origin of the training set (the green area in the gifs). Perhaps we can change it to "training set fixed origin" and "training set rolling origin". What do you think?

Best,

Javier

**Comment by KishManani** [comment]
Hi Javier,

Thanks for the clarification. In my opinion "training set origin" is still ambiguous because it doesn't specify whether we're referring to the start or end of the training period. 

Perhaps a sentence in the documentation clarifying which origin we're referring to (start of training, end of training, forecast start date or origin) would help. 

Thanks,
Kishan
